pandas/core/indexes/multi.py

```diff
@@ -2601,7 +2601,7 @@ def _partial_tup_index(self, tup, side="left"):
                     loc -= 1
                 return start + section.searchsorted(loc, side=side)
 
-            idx = lev.get_loc(lab)
+            idx = lev.get_loc(lab) if not isna(lab) else -1
```

Comment:
getting on a level should itself return -1, so why is this needed?

Reply:
@jreback  Are you sure? Because  where code will be changed is in indexes.base or _libs.index. So effect of change quite a big.